cnf-certification-test/certification/suite.go

```diff
@@ -202,3 +206,29 @@ func testContainerCertificationStatusByDigest(env *provider.TestEnvironment, val
 	}
 	testhelper.AddTestResultLog("Non-compliant", failedContainers, tnf.ClaimFilePrintf, ginkgo.Fail)
 }
+
+func testHelmVersion(env *provider.TestEnvironment) {
+	helmchartsReleases := env.HelmChartReleases
+	// Collect all of the failed helm charts
+	var compliantObjects []*testhelper.ReportObject
+	var nonCompliantObjects []*testhelper.ReportObject
+	for _, helm := range helmchartsReleases {
+		helmChartVersion := helm.Chart.Metadata.APIVersion
+		helmChartVersion = strings.Split(helmChartVersion, "+")[0]
+		helmChartVersion = strings.Split(helmChartVersion, "v")[1]
+		helmChartVersionf, err := strconv.ParseFloat(helmChartVersion, 64)
+		if err != nil {
+			logrus.Errorf("failed to parse helm %s version: %v", helm.Chart.Name(), err)
+		}
+		if helmChartVersionf > 0 && helmChartVersionf < 3 {
+			reportObject := testhelper.NewReportObject("Helm Chart need to be v3 and not v2 which is not supported due to security risks associated with Tiller", testhelper.HelmChart, false).AddField(testhelper.ChartName, helm.Name)
```

Comment:
text improvement: 
"Helm Chart need to be v3 and not v2 which is not supported due to security risks associated with Tiller" ->
fmt.Sprintf("This Helm Chart is v%d but needs to be v3 due to the security risks associated with Tiller", helmChartVersionf )